ui/apps/everest/src/pages/database-form/database-form-body/steps/first/first-step.test.tsx

```diff
@@ -65,3 +66,11 @@ describe('First Step', () => {
 
   // TODO it should be disabled if monitoring instances list length <=0 or undefined
 });
+
+describe('utils', () => {
```

Comment:
I think we should do the following:
- move "humanizeDbType" out of the database-form. This is a generic function that we can maybe move to utils > db
- the tests shouldn't be here, but also I don't think it's of any use to test this, given all the function does is to do a map lookup and return a string